build/openresty/BUILD.openresty.bazel

```diff
@@ -218,10 +295,41 @@ configure_make(
         "@lua-resty-lmdb//:all_srcs",
         "@lua-resty-events//:all_srcs",
         "@openresty_binding//:all_srcs",
+        "@ngx_wasm_module//:all_srcs",
+        "@v8//:all_srcs",
+        "@wasmer//:all_srcs",
+        "@wasmtime//:all_srcs",
+        "@openresty//:wasmx_v8_ar",
     ],
     configure_command = "configure",
     configure_in_place = True,
     configure_options = CONFIGURE_OPTIONS,
+    env = select({
+        "@kong//:wasmx_v8": {
+            "NGX_WASM_RUNTIME": "v8",
+            "NGX_WASM_RUNTIME_LIB": "$$EXT_BUILD_ROOT$$/external/v8/lib",
+            "NGX_WASM_RUNTIME_INC": "$$EXT_BUILD_ROOT$$/external/v8/include",
```

Comment:
I think the same actually applies to the other `_LIB`, `_INC` env variables? Since they are already specified above as `--with-cc-opt` and `--with-ld-opt`. There is nothing else that setting these env variables will do, it's only a convenience method instead of these `--with-xx-opt` flags. Only `NGX_WASM_RUNTIME` is needed, unless I missed something larger about how this is invoked in Bazel. But try and remove `RUNTIME_{INC,LIB,LD_OPT}` and it should still all work as-is.